lib/chef/knife/winrm_knife_base.rb

```diff
@@ -46,6 +54,8 @@ def validate_options!
           def configure_session
             validate_options!
             resolve_session_options
+            load_windows_specific_gems if @session_opts[:transport] == :sspinegotiate
+            warn_no_ssl_peer_verification if @session_opts[:no_ssl_peer_verification]
```

Comment:
What is the effect if you leave the warning out entirely until that fix? We've learned recently that excessive warnings can be pretty devastating to user experience.